server/routes/api.js

```diff
@@ -0,0 +1,49 @@
+const express = require("express")
+const router = express.Router()
+const request = require('request')
+const City = require('./../model/City')
+router.get('/city/:cityName', function (req, res) {
+    const city = req.params.cityName
+    request(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&APPID=7bf665b828fdfc577cc7819ba13ac560`, function (err, response) {
+        const dataParsed = JSON.parse(response.body)
+        console.log("getting city info")
+    if (dataParsed.cod === "404") {
```

Comment:
look into `try{} catch{}` error handling. That is the more appropriate way of dealing with errors.